server/test/data/temp-outfitter-test-data.es6

```diff
@@ -15,83 +15,147 @@ module.exports = {
       data: [37, 80, 68, 70, 45, 49, 46, 51, 13, 37, 226, 227, 207]
```

Comment:
do you want to call this file `temp-outfitter-factory.es6` to be more consistent with the `noncommericial-factory.es6`